examples/controllers/.cljstyle

```diff
@@ -0,0 +1,14 @@
+{:rules
```

Comment:
Isn't it enough to have this file at the top level? I don't see what it adds here.